integration/integration.fsproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
If possible, I think that the AppVeyor instance should be upgraded instead of downgrading the projects.

Reply:
Good point. Copied the strategy applied in Zaid-Ajaj/Feliz#465